tests/serverDescription/serverDescription-debug-001.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\ServerDescription debug output
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_not_clean(); ?>
```

Comment:
IIRC, this is only needed if the test is going to read/write data. Since you're just selecting a server here, we don't need to concern ourselves with the collection under test being empty.

Reply:
Ok, the same SKIPIF is included in `server-debug.phpt` - should I make a ticket to remove that?

Reply:
You can just edit the file in GitHub and generate a PR (instead of directly committing). I don't think we need to track that with a ticket.

Reply:
https://github.com/mongodb/mongo-php-driver/pull/1242

Reply:
Same comment as https://github.com/mongodb/mongo-php-driver/pull/1242#discussion_r680176434

I was only suggesting that `skip_if_not_clean` be removed.